state_machine/applications/flight/lib/logs.py

```diff
@@ -1,37 +1,40 @@
-from pycubed import cubesat
-from state_machine import state_machine
 import struct
 import os
 try:
-    from ulab.numpy import array
+    from ulab.numpy import array, nan
 except ImportError:
-    from numpy import array
+    from numpy import array, nan
+from pycubed import cubesat
+from state_machine import state_machine
 
 
-beacon_format = 'b' + 'f' * 11  # 1 char + 11 floats
+beacon_format = 3 * 'B' + 'H' + 'f' * 11  # 3 uint8 + 1 uint16 + 11 float32 = 49 bytes
```

Comment:
Annoyingly this will be 52 bytes because of weird c struct padding. 
You can verify this with:
```python3
import struct
format = 3*'B'+'H'+'f'*11
x=struct.pack(format, 0,0,0,0,0,0,0,0,0,0,0,0,0,0,0)
len(x)
```
returns 52.

I think this still fits, but worth noting

Reply:
fits into one packet*

Reply:
Ah, makes sense - the c struct will be word-aligned, so there's 1 extra byte along with the three chars and 2 extra bytes with the short.